external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopVillainsIT.java

```diff
@@ -45,7 +48,7 @@
     @Container(image = "${postgresql.13.image}", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
     static PostgresqlService database = new PostgresqlService();
 
-    @GitRepositoryQuarkusApplication(repo = "https://github.com/quarkusio/quarkus-workshops.git", contextDir = "quarkus-workshop-super-heroes/super-heroes/rest-villain", mavenArgs = "-Dquarkus.package.type=uber-jar -DskipTests")
+    @GitRepositoryQuarkusApplication(repo = "https://github.com/quarkusio/quarkus-workshops.git", contextDir = "quarkus-workshop-super-heroes/super-heroes/rest-villains", branch = "3d3425a15daacf1c774cb7f5bc24228c4a623256", mavenArgs = "-Dquarkus.package.type=uber-jar -DskipTests -Dquarkus.platform.group-id=${QUARKUS_PLATFORM_GROUP-ID} -Dquarkus.platform.version=${QUARKUS_VERSION}")
```

Comment:
[3d3425a15daacf1c774cb7f5bc24228c4a623256](https://github.com/quarkusio/quarkus-workshops/commit/3d3425a15daacf1c774cb7f5bc24228c4a623256) ... why commit from October 2021

There are newer commits in https://github.com/quarkusio/quarkus-workshops/commits/main

Reply:
Because in the latest versions of this application we have a dependency that we can't resolve with the current implementation of Quarkus Test Framework /  `@GitRepositoryQuarkusApplication` annotation.

https://github.com/quarkusio/quarkus-workshops/blob/main/quarkus-workshop-super-heroes/super-heroes/rest-villains/pom.xml#L98

Basically, we will need to compile the whole project first, in order to generate some internal artifacts, that then later are going to be used by `rest-villains`.

Reply:
Related to: https://github.com/quarkus-qe/quarkus-test-framework/issues/432